app/uk/gov/hmrc/plasticpackagingtax/registration/views/liability_weight_page.scala.html

```diff
@@ -1,70 +1,88 @@
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *@
-
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+@import uk.gov.hmrc.govukfrontend.views.html.components.{GovukButton, GovukInput, govukFieldset}
 @import uk.gov.hmrc.govukfrontend.views.html.helpers.formWithCSRF
-@import uk.gov.hmrc.govukfrontend.views.html.components.GovukInput
-@import uk.gov.hmrc.govukfrontend.views.html.components.GovukButton
-@import uk.gov.hmrc.govukfrontend.views.html.components.govukFieldset
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.Title
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.BackButton
 @import uk.gov.hmrc.plasticpackagingtax.registration.controllers.{routes => pptRoutes}
 @import uk.gov.hmrc.plasticpackagingtax.registration.forms.LiabilityWeight
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.paragraphBody
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.sectionHeader
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.errorSummary
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.saveButtons
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.{errorSummary, paragraphBody, saveButtons, sectionHeader}
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.{BackButton, Title}
 
 @this(
-  govukLayout: main_template,
-  paragraphBody: paragraphBody,
-  govukButton: GovukButton,
-  govukInput: GovukInput,
-  sectionHeader: sectionHeader,
-  govukFieldset: govukFieldset,
-  errorSummary: errorSummary,
-  formHelper: formWithCSRF,
-  saveButtons: saveButtons,
+        govukLayout: main_template,
+        paragraphBody: paragraphBody,
+        govukButton: GovukButton,
+        govukInput: GovukInput,
+        sectionHeader: sectionHeader,
+        govukFieldset: govukFieldset,
+        errorSummary: errorSummary,
+        formHelper: formWithCSRF,
+        saveButtons: saveButtons,
+        govukInsetText: GovukInsetText,
 )
 @(form: Form[LiabilityWeight])(implicit request: Request[_], messages: Messages)
 
-@totalWeightField = @{form("totalKg")}
+@totalWeightField = @{
+    form("totalKg")
```

Comment:
Any chance we can change this to `value` instead of `totalKg` for consistency across all single field screens that we control? No problem if there's some technical reason as to why we can't...

Reply:
Hmmm... not really sure what the impact of that would be. I'll have to look into it and come back to you.